processing-status-api-function-app/src/main/kotlin/gov/cdc/ocio/processingstatusapi/functions/status/GetStatusFunction.kt

```diff
@@ -97,75 +79,6 @@ class GetStatusFunction(
         val excludedSpanTags = listOf("spanMark", "span.kind", "internal.span.format", "otel.library.name")
```

Comment:
excludedSpanTages can be removed.  Since it is the only element, you can remove the entire companion object.

Reply:
Completed